pyk/src/pyk/kast.py

```diff
@@ -812,7 +812,7 @@ def let(
     ) -> 'KProduction':
         sort = sort if sort is not None else self.sort
         items = items if items is not None else self.items
-        klabel = klabel if klabel is not None else self.klabel
+        klabel = klabel if klabel is not None else self.klabel  # TODO figure out a way to set klabel to None
```

Comment:
Should this TODO be resolved?

Reply:
@tothtamas28 ^

Reply:
This can be resolved in a later PR. We will need to perform similar changes for some other classes as well.